tasks/e2e.sh

```diff
@@ -135,7 +138,7 @@ npm install $cli_path
 
 # Install the app in a temporary location
 cd $temp_app_path
-create_react_app --scripts-version=$scripts_path test-app
+create_react_app --scripts-version=$scripts_path --template=$root_path/packages/react-scripts/fixtures/kitchensink test-app
```

Comment:
I don't like that we have completely replaced the simple e2e test with the kitchensink test.
I would prefer to copy and paste the original `e2e` test to `e2e-kitchensink` and make changes there.

This way we have a simple "fail fast" version that catches simple errors, and a more sophisticated template for testing of individual features.

Could you please separate them and make `e2e` script in `package.json` run both e2e tests, first simple, and then the complex one?

Don’t worry about duplication, it’s fine.